core/application-crypto/src/ed25519.rs

```diff
@@ -21,6 +21,7 @@ use crate::{RuntimePublic, KeyTypeId};
 pub use primitives::ed25519::*;
 
 mod app {
+	use crate::Vec;
```

Comment:
No this is not correct, please fix the macro to include `Vec` with an absolute path.

Reply:
ok